File: tree-hugger/lib/tests/fixtures/imports.go
// Test fixture for Go import extraction
package main

// Simple import
import "fmt"

// Grouped imports
import (
	"os"
	"strings"
)

// Aliased import
import alias "path/filepath"

// Blank import (for side effects)
import _ "net/http/pprof"

func main() {
	fmt.Println("hello")
	alias.Join("a", "b")
}

File: tree-hugger/lib/tests/fixtures/imports.py
# Test fixture for Python import extraction

# Simple import
import os
import sys

# From import
from typing import Optional, List

# Aliased import
import numpy as np

# From import with alias
from collections import OrderedDict as OD

# Relative import (in a package context)
# from . import utils
# from ..models import User

def main():
    print(os.getcwd())
    items: List[str] = []
